src/basic-syntax/static-and-const.md

```diff
@@ -1,10 +1,10 @@
 # Static and Constant Variables
 
-Global state is managed with static and constant variables.
+Static and constants are two kind of variables with special constraints.
 
 ## `const`
 
-You can declare compile-time constants:
+Constants are values created at compile time. They are inherently more efficient because they don't need to be computed at runtime and the compiler can inline them and apply special optimizations that would not be possible with a larger set of values:
```

Comment:
You're turning single-line sentences (like the ones found on a slide deck) into multi-line paragraphs. That's an anti-pattern and something I'm putting together a style guide (#591) to try and avoid.

Basically, please try looking at the slide with the "aspect ratio helper" enabled in your `book.toml` file. I'm pretty sure it overflow the height by a lot now.